packages/azure-themes/src/azure/styles/Callout.styles.ts

```diff
@@ -1,17 +1,13 @@
 import { ICalloutContentStyleProps, ICalloutContentStyles } from 'office-ui-fabric-react/lib/Callout';
 import { Depths } from '../AzureDepths';
-import * as StyleConstants from '../Constants';
 
 export const CalloutContentStyles = (props: ICalloutContentStyleProps): Partial<ICalloutContentStyles> => {
   const { theme } = props;
   const { semanticColors } = theme;
 
   return {
     root: {
-      borderColor: semanticColors.inputBorder,
-      borderStyle: StyleConstants.borderSolid,
-      borderWidth: StyleConstants.borderWidth,
-      boxShadow: Depths.depth8,
```

Comment:
This also affect the `Callout` component, but I didn't see it mentioned in the PR description? Is this change intended? 
 
| Before | After |
| --- | --- |
| ![Screen Shot 2020-10-07 at 1 39 15 PM](https://user-images.githubusercontent.com/2100222/95367298-96a43480-08a2-11eb-9c86-db8068fbc3d4.png) | ![Screen Shot 2020-10-07 at 1 38 05 PM](https://user-images.githubusercontent.com/2100222/95367302-973ccb00-08a2-11eb-952c-031717adcc19.png) |

Reply:
@wsmd great point, I left out that screenshot. All of our dialogs / dropdowns have no border now. I will reduce the dropshadow and include image:


## Figma design

<img src="https://user-images.githubusercontent.com/30805892/95370782-2b506780-088e-11eb-92d0-a1e7970906ca.png"  height="300" />